src/atopile/front_end.py

```diff
@@ -833,27 +836,27 @@ def _new_node(
             return result
 
         # This should never happen
-        raise ValueError(f"Unknown item type {item}")
+        raise ValueError(f"Unknown item type `{item}`")
 
+    @contextmanager
     def _init_node(
-        self,
-        node_type: ap.BlockdefContext | Type[L.Node],
-    ) -> L.Node:
+        self, node_type: ap.BlockdefContext | Type[L.Node]
```

Comment:
@iopapamanoglou this is an interesting pattern. Not 100% one way or another on how much I like it, but seems like it or `super()` calls could help in fabll too?